test.mjs

```diff
@@ -21,7 +21,7 @@ function testContext({any, html, css, gql, md, sql}, name) {
 			assert.equal(any, sql);
 		});
 
-		it('code-tags', () => {
+		it('code-tag', () => {
```

Comment:
It was meant to be called `code-tags` but there's already `codetags` on npm and it's not allowed.